cartridge-cli.lua

```diff
@@ -1018,17 +1087,10 @@ local function remove_ignored(destdir)
     end
 end
 
-local function form_distribution_dir(source_dir, destdir, app_name, app_version)
+local function form_distribution_dir(source_dir, destdir, app_name, app_version, opts)
```

Comment:
Instead of passing options to this function, you may just factor out the code that is responsible for copying and cleaning up the source tree to a separate function, and then call that function from here. 

Think about it this way: if you try to write documentation for this function, you will struggle with describing what exactly it does.

Reply:
Moved building application and copying tarantool binaries to `build_application` and `copy_taranool_binaries` functions.
`generate_version_file` function is now called outside of `form_distribution_dir ` (to be called after `tarantoolctl rocks make`).